CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Initial connector which outputs all analytics into a single elasticsearch index
 - Enabled the ES connecter to save all output to a rolling index rather than a static index
+- Add support to the Prometheus pump to congigure metrics families from exposition 
```

Comment:
Export would be more natural english than exposition. Thanks!

Reply:
I picked up the term from OpenTelemetry/[OpenMetrics](https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md) documentation. Exposition is the term used to describe a particular scrape cycle in the prometheus model.

Happy to change though.

Reply:
Very interesting thanks for the context. It looks like that term is used widely enough so happy to keep it.